src/containers/NewTask.tsx

```diff
@@ -9,118 +9,91 @@ import { connect } from 'react-redux'
 import { Actions } from 'src/actions'
 import { NavigationScreenProp } from 'react-navigation'
 import { NavigationRoute } from 'react-navigation'
-import { State } from 'src/types/global'
 import { Dispatch } from 'redux'
-import { changeFormAction } from 'src/actions/types'
+import { NavigationScreenComponent } from 'react-navigation'
 
 export interface Props {
   navigation: NavigationScreenProp<NavigationRoute>
   title: string
   description: string
   addTask: (title: string, description: string) => void
-  onChangeForm: (title: string, description: string) => void
-  resetForm: () => void
 }
 
-class LayoutNewTask extends Component<Props> {
-  static navigationOptions = ({ navigation }: any) => {
-    const { params = {} } = navigation.state
-    const { addTask, title, description } = params
-    const add = () => {
-      if (addTask !== undefined) {
-        addTask(title, description)
-        navigation.navigate(Route.Home)
-      }
-    }
-    const buttonSave = (
-      <TouchableOpacity onPress={add}>
-        <Text style={newTaskStyles.buttonSave}>{strings.save}</Text>
-      </TouchableOpacity>
-    )
-
-    return {
-      title: Route.NewTask,
-      headerStyle: {
-        backgroundColor: colors.clearBlue,
-      },
-      headerTintColor: colors.white,
-      headerTitleStyle: homeStyles.text,
-      headerRight: buttonSave,
-    }
-  }
+const LayoutNewTask: NavigationScreenComponent<{}, {}, Props> = (
```

Comment:
I see that this could be typed like 
NavigationScreenComponent<
    Params = NavigationParams,
    Options = {},
    Props = {}
  >

so in your case this should be `NavigationScreenComponent<NavigationParams, {}, Props>`